lib/sourcebit.js

```diff
@@ -169,12 +170,13 @@ class Sourcebit {
     let queue = Promise.resolve(initialData);
 
     this.pluginBlocks.forEach(({ _isBootstrapped, options }, index) => {
-      if (!_isBootstrapped) return data;
+      if (!_isBootstrapped) {
+        return initialData;
```

Comment:
Well, when a plugin is not bootstrapped we return `initialData` from the `forEach` method, so that return value just ends up being discarded – it's only the return value of `queue` that matters, but in this case we don't even get there. I'll change L174 to just return `undefined`.